src/components/ui.css.ts

```diff
@@ -187,28 +211,32 @@ export const margin = styleVariants(
   },
   (margin) => ({ margin })
 )
-margin.left = styleVariants(
+
+export const marginLeft = styleVariants(
   {
     ...theme.space,
     auto: "auto",
   },
   (marginLeft) => ({ marginLeft })
 )
-margin.right = styleVariants(
+
+export const marginRight = styleVariants(
   {
     ...theme.space,
     auto: "auto",
   },
   (marginRight) => ({ marginRight })
 )
-margin.top = styleVariants(
+
+export const marginTop = styleVariants(
   {
     ...theme.space,
     auto: "auto",
   },
   (marginTop) => ({ marginTop })
 )
-margin.bottom = styleVariants(
+
+export const marginBottom = styleVariants(
```

Comment:
Not a problem, but this just feels less elegant than grouping into an object, but keeps things more consistent

Reply:
Yeah I changed this one because of the TS intellisense warning:

<img width="729" alt="Screen Shot 2022-03-24 at 7 37 57 PM" src="https://user-images.githubusercontent.com/1790506/160027228-48407e30-9fc6-4e26-a39d-50462e3b0917.png">